<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>demarcateDlg</class>
 <widget class="QDialog" name="demarcateDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1211</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TCP坐标点:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>激光头坐标点:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>标定模式:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>450</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>眼在手上</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>490</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>眼在手外</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>410</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>添加TCP点</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>470</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>替换TCP点</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>410</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>删除TCP点</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>410</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>添加激光头点</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>470</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>替换激光头点</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>410</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>删除激光头点</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>410</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>计算标定结果</string>
   </property>
  </widget>
  <widget class="QLabel" name="a22_label">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>500</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>a33</string>
   </property>
  </widget>
  <widget class="QLabel" name="a01_label">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>420</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>a12</string>
   </property>
  </widget>
  <widget class="QLabel" name="a11_label">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>460</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>a22</string>
   </property>
  </widget>
  <widget class="QLabel" name="a02_label">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>420</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>a13</string>
   </property>
  </widget>
  <widget class="QLabel" name="a20_label">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>500</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>a31</string>
   </property>
  </widget>
  <widget class="QLabel" name="a00_label">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>420</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>a11</string>
   </property>
  </widget>
  <widget class="QLabel" name="a10_label">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>460</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>a21</string>
   </property>
  </widget>
  <widget class="QLabel" name="a21_label">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>500</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>a32</string>
   </property>
  </widget>
  <widget class="QLabel" name="a12_label">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>460</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>a23</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>500</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>标定矩阵:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>470</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>标定精度:</string>
   </property>
  </widget>
  <widget class="QLabel" name="err">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>470</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>470</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>清空TCP点</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>470</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>清空激光头点</string>
   </property>
  </widget>
  <widget class="QLabel" name="t00_label">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>420</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>t11</string>
   </property>
  </widget>
  <widget class="QLabel" name="t10_label">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>460</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>t21</string>
   </property>
  </widget>
  <widget class="QLabel" name="t20_label">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>500</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>t31</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="record">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>1191</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>调试信息:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="robposlist">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>391</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="leaserposlist">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>791</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="t01_label">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>420</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>t12</string>
   </property>
  </widget>
  <widget class="QLabel" name="t02_label">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>420</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>t13</string>
   </property>
  </widget>
  <widget class="QLabel" name="t11_label">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>460</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>t22</string>
   </property>
  </widget>
  <widget class="QLabel" name="t12_label">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>460</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>t23</string>
   </property>
  </widget>
  <widget class="QLabel" name="t21_label">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>500</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>t32</string>
   </property>
  </widget>
  <widget class="QLabel" name="t22_label">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>500</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>t33</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
